include/glaze/core/common.hpp

```diff
@@ -239,10 +239,16 @@ namespace glz
          (!std::same_as<std::nullptr_t, T> && std::constructible_from<std::string_view, std::decay_t<T>>) ||
          array_char_t<T>;
 
+      // static string
+      template <class T>
+      concept static_str_t =
+       str_t<T> && !string_view_t<T> && has_assign<T> && is_static<T>;
```

Comment:
I think we need to add `resizable<T> && has_data<T>`. It has fixed capacity, but not fixed size.